lib/remote/filterutility.cpp

```diff
@@ -166,17 +166,43 @@ void FilterUtility::CheckPermission(const ApiUser::Ptr& user, const String& perm
 				else
 					*permissionFilter = new LogicalOrExpression(std::unique_ptr<Expression>(*permissionFilter), std::unique_ptr<Expression>(fexpr));
 			}
+
+			break;
```

Comment:
This is a bad idea. Figure out and explain in your own words why.

Reply:
Is it probably due to the fact that the same permission might be used with different filters repeatedly? I don't know, why should one even do something like that?